snapshot/lib/snapshot/fixes/hardware_keyboard_fix.rb

```diff
@@ -5,8 +5,6 @@ module Fixes
 
     class HardwareKeyboardFix
       def self.patch
-        Snapshot.kill_simulator # First we need to kill the simulator
```

Comment:
Better to kill more often than not enough 😉 But yah, we can probably get rid of some